simplq/src/components/pages/Join/index.jsx

```diff
@@ -45,6 +46,7 @@ export default (props) => {
     <div>
       <HeaderSection queueName={queueStatusResponse.queueName} history={props.history} />
       <div className={styles['main-content']}>
+        <QueueStats queueStatus={queueStatusResponse} />
```

Comment:
In this or later pr, can we do something like:
```
<div className={styles['main-content']}>
  <div className={styles['section']}>
        <p className={styles['message']}>Please enter your contact details to join this queue</p>
        <JoinQueueForm
          queueId={queueId}
          joinQueueHandler={joinQueueHandler}
          buttonText="Join Queue"
        />
  </div>
  <div className={styles['section']}>
     <QueueStats queueStatus={queueStatusResponse} />
     < svg here >
  </div>
</div>
```

It would then look roughly like in our initial design:

![image](https://user-images.githubusercontent.com/11256376/105640974-7aef8900-5ea7-11eb-98d7-37f94b919306.png)

You can ignore the background color of the right side, or have it, upto you. On winder screens the two sections can come side by side, but on mobile, they can then flex-wrap so that the stats go below.

What do you think?

Reply:
This was the Idea. I just extracted the existing code that can be reused. Should we also render different background color based on status (ACTIVE, PAUSED, DELETED)?

Reply:
`styles['section']` loks like a very good solution. Should be designed and reused in all pages since currently pages don't look good on big screen in different sizes.

![Screenshot 2021-01-24 at 22 41 26](https://user-images.githubusercontent.com/10821730/105658875-7b7f2400-5e95-11eb-890e-4e1948955875.png)

![Screenshot 2021-01-24 at 22 42 16](https://user-images.githubusercontent.com/10821730/105658966-a36e8780-5e95-11eb-901d-9f6b1dae79b2.png)

Reply:
Different colors based on status is not a bad idea. But I'm worried it would mess with the color scheme of the site. We're using this tint of blue everywhere and having a single page which is predominantly a different color seems out of place. If you have some ideas, could you post pictures of the design, so we can deliberate?

Reply:
Agreed. Let's not change the background colour entirely, but it will we good UX to highlight attention to PAUSED and DELETED as @maaverik said.

Reply:
> styles['section'] loks like a very good solution. Should be designed and reused in all pages since currently pages don't look good on big screen in different sizes.

This is a great idea, we should try to do this. If you can raise an issue regarding this, we can come up with a proposal and take this up.

Reply:
I would leave it at this state for now. I will open an issue with some suggestions and we can go from there.